### ISSUE TITLE
customizable detector

### ISSUE THREAD

**Issue by Adas-22** [studio-minus/ppg-bugs-and-requests#2261]
**Feature description**
Customizable detector is a new detector that will have the "object selection" function. When this function is enabled, the first object that touches the laser will be able to trigger the detector. The second implementation of the function: objects of the same type will be able to trigger the detector.

**Feature purpose**
Such detectors will be very useful for some mechanisms. 

**Additional context**
—

**Comment by DindinYT37** [comment]
Duplicate of or Related to:
* #588
* #598
* #635
* #636
* #685
* #853
* #988
* #1065
* #1117
* #1205
* #1795 
* #1948 
* #1954 
* #1963 
* #2108

**Comment by Adas-22** [comment]
> Duplicate of or Related to:
> * #588
> * #598
> * #635
> * #636
> * #685
> * #853
> * #988
> * #1065
> * #1117
> * #1205
> * #1795 
> * #1948 
> * #1954 
> * #1963 
> * #2108 
> 

Okay. Only this has not been added yet, which means that it is worth editing/duplicating it somehow for zooi's attention.

**Comment by DindinYT37** [comment]
> Okay. But this has not been added yet, which means that it is worth editing/duplicating it somehow for zooi's attention.

Not at all, that's completely contradictory to what you're trying to reach. This is an issue-board, not a forum - reposting the same thing over and over again will just make it less likely to get added. Though I cannot be sure since I don't have a connection with zooi of some sort, it's usually done as a form of punishment because of the constant demand.

**Comment by Adas-22** [comment]
> > Okay. But this has not been added yet, which means that it is worth editing/duplicating it somehow for zooi's attention.
> 
> Not at all, that's completely contradictory to what you're trying to reach. This is an issue-board, not a forum - reposting the same thing over and over again will just make it less likely to get added. Though I cannot be sure if zooi would do this since I don't have a connection with zooi of some sort - usually game developers ignore constantly reposted requests as a form of punishment.

Well, I don't really care about that. I have expressed my idea and perhaps I will refine it.

**Comment by DindinYT37** [comment]
> > > Okay. But this has not been added yet, which means that it is worth editing/duplicating it somehow for zooi's attention.
> > 
> > 
> > Not at all, that's completely contradictory to what you're trying to reach. This is an issue-board, not a forum - reposting the same thing over and over again will just make it less likely to get added. Though I cannot be sure if zooi would do this since I don't have a connection with zooi of some sort - usually game developers ignore constantly reposted requests as a form of punishment.
> 
> Well, I don't really care about that. I have expressed my idea and perhaps I will refine it.

Well, if you don't care about that, then you also don't have to care when this gets closed.

**Comment by Adas-22** [comment]
> > > > Okay. But this has not been added yet, which means that it is worth editing/duplicating it somehow for zooi's attention.
> > > 
> > > 
> > > Not at all, that's completely contradictory to what you're trying to reach. This is an issue-board, not a forum - reposting the same thing over and over again will just make it less likely to get added. Though I cannot be sure if zooi would do this since I don't have a connection with zooi of some sort - usually game developers ignore constantly reposted requests as a form of punishment.
> > 
> > Well, I don't really care about that. I have expressed my idea and perhaps I will refine it.
> 
> Well, if you don't care about that, then you also don't have to care when this gets closed and not added.

Well, they will close and close. What is it to mumble? So I'll make the modification myself. There are many ways to achieve your goal and this one may not be the only one

**Comment by DindinYT37** [comment]
> > > > > Okay. But this has not been added yet, which means that it is worth editing/duplicating it somehow for zooi's attention.
> > > > 
> > > > 
> > > > Not at all, that's completely contradictory to what you're trying to reach. This is an issue-board, not a forum - reposting the same thing over and over again will just make it less likely to get added. Though I cannot be sure if zooi would do this since I don't have a connection with zooi of some sort - usually game developers ignore constantly reposted requests as a form of punishment.
> > > 
> > > 
> > > Well, I don't really care about that. I have expressed my idea and perhaps I will refine it.
> > 
> > 
> > Well, if you don't care about that, then you also don't have to care when this gets closed and not added.
> 
> Well, they will close and close. What is it to mumble? So I'll make the modification myself. There are many ways to achieve your goal and this one may not be the only one

What are you even on about? If you wanted to make the mod yourself, then do it and don't waste other people's time by posting yet another duplicate on the issue-board.

**Comment by Adas-22** [comment]
> > > > > > Okay. But this has not been added yet, which means that it is worth editing/duplicating it somehow for zooi's attention.
> > > > > 
> > > > > 
> > > > > Not at all, that's completely contradictory to what you're trying to reach. This is an issue-board, not a forum - reposting the same thing over and over again will just make it less likely to get added. Though I cannot be sure if zooi would do this since I don't have a connection with zooi of some sort - usually game developers ignore constantly reposted requests as a form of punishment.
> > > > 
> > > > 
> > > > Well, I don't really care about that. I have expressed my idea and perhaps I will refine it.
> > > 
> > > 
> > > Well, if you don't care about that, then you also don't have to care when this gets closed and not added.
> > 
> > Well, they will close and close. What is it to mumble? So I'll make the modification myself. There are many ways to achieve your goal and this one may not be the only one
> 
> What are you even on about? If you wanted to make the mod yourself, then do it and don't waste other people's time by posting yet another duplicate on the issue-board.

I don't think I care about your opinion about my actions. I'll decide what I do and what I don't

**Comment by DindinYT37** [comment]
> > What are you even on about? If you wanted to make the mod yourself, then do it and don't waste other people's time by posting yet another duplicate on the issue-board.
> 
> I don't think I care about your opinion about my actions. I'll decide what I do and what I don't

This is not my opinion, it's the objective conclusion I've come to upon evaluating this issue and what you've been telling me.

**Comment by DindinYT37** [comment]
Now, let's stop cluttering this issue with your useless arguments and my redundant replies and stop this, as there's literally no use in further arguing.
